runtime/browser/xwalk_browser_main_parts.cc

```diff
@@ -27,6 +27,9 @@
 #include "xwalk/application/browser/application_system.h"
 #include "xwalk/extensions/browser/xwalk_extension_service.h"
 #include "xwalk/extensions/common/xwalk_extension_switches.h"
+#if defined(USE_GTK_UI)
```

Comment:
is this really the best FLAG? I think that it would be better with a more specific one